app/src/components/features/rules/TestThisRule/index.tsx

```diff
@@ -26,8 +28,10 @@ export const TestThisRuleRow: React.FC = () => {
   const isCurrentlySelectedRuleHasUnsavedChanges = useSelector(getIsCurrentlySelectedRuleHasUnsavedChanges);
   const currentlySelectedRuleData = useSelector(getCurrentlySelectedRuleData);
   const testThisRuleBoxRef = useRef(null);
+
   const [pageUrl, setPageUrl] = useState("");
   const [error, setError] = useState(null);
+  const [recordTestPage, setRecordTestPage] = useState<boolean>(!!isFeatureCompatible(FEATURES.TEST_THIS_RULE_SESSION));
```

Comment:
This is uneccessary as event if we send record=true to older extension, it wont still record the page.